modules/store/src/action_group_creator.ts

```diff
@@ -138,17 +138,16 @@ export function createActionGroup<
   Events extends Record<string, ActionCreatorProps<unknown> | Creator>
 >(config: ActionGroupConfig<Source, Events>): ActionGroup<Source, Events> {
   const { source, events } = config;
-
-  return Object.keys(events).reduce(
-    (actionGroup, eventName) => ({
+  return Object.entries(events).reduce((actionGroup, [eventName, event]) => {
+    return {
       ...actionGroup,
+      // TODO: fix the any type
       [toActionName(eventName)]: createAction(
         toActionType(source, eventName),
-        events[eventName]
+        event as any
```

Comment:
This is a quick-fix that we should come back at.
I wasn't able to fix without touching more types, help is appreciated here 😅
The problem is passing the correct event type to `createAction`.